app/jobs/update_user_assessment_attempt_from_class_marker_job.rb

```diff
@@ -6,6 +6,10 @@ def perform(test_id, user_id, percentage)
     assessment = Assessment.find_by!(class_marker_test_id: test_id)
     latest_attempt = assessment.latest_attempt_for(user:)
 
+    if percentage.nil?
```

Comment:
I don't think this is strictly needed, as I believe the only place this is called is from the Classmarker webhook, and that would never return without a percentage